hub-templates/base-hub/values.yaml

```diff
@@ -58,6 +58,18 @@ jupyterhub:
     chp:
       nodeSelector:
         hub.jupyter.org/pool-name: core-pool
+      networkPolicy:
+        enabled: true
+        ingress:
+        - from:
+          - namespaceSelector:
+              matchLabels:
+                name: support
+          - podSelector:
+              matchLabels:
+                hub.jupyter.org/network-access-proxy-http: "true"
+          ports:
+          - port: http
```

Comment:
I believe this section won't be needed, because http access to the autohttps and chp pods are granted by default.